src/main/java/net/dv8tion/jda/api/entities/Message.java

```diff
@@ -207,7 +207,7 @@ public interface Message extends ISnowflake, Formattable
             "(?:https?://)?" +                     // Scheme
             "(?:\\w+\\.)?" +                       // Subdomain
             "discord(?:(?:app)?\\.com" +           // Discord domain
-            "/invite|\\.gg)/(?<code>[a-z0-9-]+)" + // Path
+            "/invite|\\.gg)[/\\\\](?<code>[a-z0-9-]+)" + // Path
```

Comment:
The client doesn't allow this for `discord.gg\code` but it does for `discord.com/invite\code`

![image](https://github.com/user-attachments/assets/5c523fbf-637a-469f-a425-36137c034ba6)

Your regex also applies to `.gg\code` though